### ISSUE TITLE
Add contributor list to the readme

### ISSUE THREAD

**Issue by s-yadav** [s-yadav/iv-viewer#59]

**Comment by grlwholifts** [comment]
hi, can I work on this?

**Comment by s-yadav** [comment]
Sure, go ahead.

**Comment by grlwholifts** [comment]
could you specify the issue a bit more, exactly which contributors need to be added? 😄

**Comment by grlwholifts** [comment]
there are 3 contributors that I can see from the attached image. I guess I will need to add a separate section to the README and add their handles. Is that all?  

![image](https://user-images.githubusercontent.com/30324532/66105824-205b5b00-e5da-11e9-9c17-b3a75b81be0b.png)

**Comment by s-yadav** [comment]
Yes, I think usually it comes as API for github markdown. So the contributor list will be up to date. Have to check.

**Comment by s-yadav** [comment]
Check this. This is a github bot. 
https://github.com/all-contributors/all-contributors/

**Comment by grlwholifts** [comment]
I think, adding this [file](https://allcontributors.org/docs/en/bot/configuration) in the code would help. Afterwords, we an add the contributors using the bot commands ` add @all-contributors`. What do you think?

**Comment by s-yadav** [comment]
👍

**Comment by s4m0r4m4** [comment]
This can be closed, the README has the list of contributors
